module-bsp/board/rt1051/puretx/bsp/battery_charger/MAX77818.hpp

```diff
@@ -181,6 +187,20 @@ namespace bsp::battery_charger
         lim1600mA = 0x20, /// 1C of battery
     };
 
+    // CHG_CNFG_06
+    enum class ChargerSettingsProtection
+    {
+        Enable  = 0,
+        Disable = (0x03 << 2)
```

Comment:
This shift looks a bit weird here. I would instead use `0x0C` ;)

Reply:
I wrote it this way as it is shown like this in RM of MAX77818: https://www.analog.com/media/en/technical-documentation/data-sheets/MAX77818.pdf, see p. 71, `CHG_CNFG_06 Register Bit Description (0xBD)` - it states that `0x03 = write capability is unlocked`, and the value is set on bits 2 and 3, so I just left-shifted the value copied from the docs :P If you prefer I'll change this ;)

Reply:
You don't need to change this then ;) Firstly I thought that the explicit value is 0x0C.